requirements/tests.in

```diff
@@ -0,0 +1,6 @@
+-r base.txt
+pylint
+pytest
+pre-commit
```

Comment:
Nit: both `pylint` and `pytest` would actually belong to `dev.in`. Actually we discussed a while ago to remove `tests.in` and only use `dev.in` or something similar (you can check ticket 3679). This might be a good moment to do it for the SDK. What do you think?

Reply:
I marked it as a nit because there's the other open ticket that will deal with it, but I think it might help to do it now if you have the time.

Reply:
Done in https://github.com/gridsingularity/gsy-e-sdk/pull/194/commits/d4a9ea3368fc1e064f17177aa4c4f4ea124cb48b